.travis.yml

```diff
@@ -3,11 +3,21 @@ language: go
 go:
   - "1.8"
 
+addons:
+  postgresql: "9.5"
+
+env:
+  - TEST_DB_NAME=starttls_test
+
 install:
   - go get -u github.com/golang/lint/golint
   - go get ./...
 
+before_script:
+  - psql -c 'CREATE DATABASE starttls_test;' -U postgres
+  - psql -c "ALTER USER postgres WITH PASSWORD 'postgres';" -U postgres
+  - psql starttls_test < db/scripts/init_tables.sql
```

Comment:
Unrelated to this PR, but this line reminded me that we have to think about how we want db migrations to happen: #90